gen.go

```diff
@@ -1591,7 +1689,29 @@ func (t *{{ .Name}}) UnmarshalCBOR(r io.Reader) (err error) {
 		} else {
 			f.Name = "t." + f.Name
 		}
-		fmt.Fprintf(w, "\t// %s (%s) (%s)\n", f.Name, f.Type, f.Type.Kind())
+
+		genericParam := gti.toGenericTypeParam(f.Type.String())
+		if genericParam != "" {
+			data := map[string]interface{}{
+				"Name":         f.Name,
+				"GenericParam": genericParam,
+			}
```

Comment:
Can we pre-compute this and store it in the field?

Reply:
yes, we can